Lib/http/server.py

```diff
@@ -117,13 +117,25 @@ class HTTPServer(socketserver.TCPServer):
     allow_reuse_address = True    # Seems to make sense in testing environment
     allow_reuse_port = True
 
+    def __init__(self, *args, response_headers=None, **kwargs):
+        self.response_headers = response_headers
+        super().__init__(*args, **kwargs)
+
     def server_bind(self):
         """Override server_bind to store the server name."""
         socketserver.TCPServer.server_bind(self)
         host, port = self.server_address[:2]
         self.server_name = socket.getfqdn(host)
         self.server_port = port
 
+    def finish_request(self, request, client_address):
+        """Finish one request by instantiating RequestHandlerClass."""
+        args = (request, client_address, self)
+        kwargs = {}
+        response_headers = getattr(self, 'response_headers', None)
+        if response_headers:
+            kwargs['response_headers'] = self.response_headers
+        self.RequestHandlerClass(*args, **kwargs)
```

Comment:
```suggestion
        kwargs = {}
        if hasattr(self, 'response_headers'):
            kwargs['response_headers'] = self.response_headers
        self.RequestHandlerClass(request, client_address, self, **kwargs)
```

Reply:
@picnixz I made this requested change in 77b5fff86aa309120c49c206e795765161753dab. Note though that now `HTTPServer` will pass `response_headers` to the `RequestHandler` class even if `response_headers` is `None` or `{}`. Most `RequestHandler` implementation constructor implementations don't take this argument, so in order for this to work I added `**kwargs` as an argument to `BaseRequestHandler.__init__`. My earlier implementation was trying to prevent this, to keep any changes to only `http/server.py` and not need to touch anything in `socketserver.py`. Perhaps the `**kwargs` addition is ok, or I'm open to other solutions if we can think of better ones.